hack/tools/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/metal3-io/ip-address-manager/hack/tools
 
-go 1.22
+go 1.22.0
```

Comment:
Maybe we should do manual bump, so we can set this to `go 1.22.3` and avoid the toolchain directive getting added.